src/Web/wwwroot/app/settings.js

```diff
@@ -1,2 +1,2 @@
 angular.module("bit")
-.constant("appSettings", {"rememberedEmailCookieName":"bit.rememberedEmail","version":"1.2.2","environment":"Development","apiUri":"http://localhost:4000"});
+.constant("appSettings", {"rememberedEmailCookieName":"bit.rememberedEmail","version":"1.3.0","environment":"Development","apiUri":"http://localhost:4000"});
```

Comment:
Sorry, wasn't sure if I was supposed to build or not but I did have to change it to production because I did not have a copy of the server on my computer, and I guess it tampered with this file.